app/src/main/java/com/sopt/now/ui/login/LoginActivity.kt

```diff
@@ -71,29 +47,59 @@ class LoginActivity : BindingActivity<ActivityLoginBinding>(R.layout.activity_lo
         }
     }
 
-    private fun validateLogin(): Boolean {
-        return validateId() && validatePassword()
+    private fun initButton() {
+        binding.btnLoginToSignup.setOnClickListener {
+            initSignUpBtnClickListener()
+        }
+
+        binding.btnLogin.setOnClickListener {
+            loginButtonEvent()
+        }
+    }
+
+    private fun initObserver() {
+        viewModel.liveData.observe(this) { state ->
+            when (state) {
+                is UiState.Loading -> {
```

Comment:
Loading 상태에는 무슨 일을 하나요?

Reply:
```suggestion
                is UiState.Loading -> Unit
```
따로 상태를 설정한게 없다면 Unit으로 작성해도 됩니다

Reply:
감사합니다! 아직 Uistate 활용법이 익숙하지 않네요..
비워두고 나중에 스켈레톤 UI 띄우는 코드가 필요한가 하고 고민했었네요